modules/gitops-disable-dex-using-spec-sso.adoc

```diff
@@ -0,0 +1,25 @@
+// Module is included in the following assemblies:
+//
+// * configuring-sso-for-argo-cd-using-dex
+
+:_content-type: PROCEDURE
+[id="gitops-disable-dex-using-spec-sso_{context}"]
+= Enabling/Disabling Dex using .spec.sso
+
+In {gitops-title} v1.6.0 `.spec.dex` is deprecated and support will be removed in v1.9.0. Dex can be configured through `.spec.sso.dex` parameter.
+
+.Procedure
+
+* To enable Dex, set the `.spec.sso.provider: dex` parameter in the *YAML* resource of the Operator:
+
++
+[source,yaml]
+----
+spec:
+  sso:
+    provider: dex
+    dex:
+      openShiftOAuth: true
+----
++
+* To disable dex, remove the `spec.sso` from the Argo CD custom resource or switch to a different SSO provider.
```

Comment:
```suggestion
. Otherwise, to disable Dex, either remove the `spec.sso` element from the Argo CD custom resource or specify a different SSO provider.
```

Reply:
@rolfedh not sure why to use the "Otherwise". Enabling Dex and disabling dex are two different things. Skipping that part, implementing rest of the suggestion.